datastore-benchmarks/benchmarks/transactionsize.js

```diff
@@ -0,0 +1,20 @@
+function objectSize(o) {
```

Comment:
This function seems arbitrary. You encode booleans as four bytes, strings apparantly as utf16, but when you encode objects, you don't encode the key-properties.

You could use msgpack to get some decent results and proper results for the object sizes. Or you simply use JSON encoding as Automerge is currently doing pre v1. These results are not comparable.